Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -251,7 +251,7 @@ struct CombinatorialKalmanFilterResult {
   Result<void> result{Result<void>::success()};
 
   // TODO place into options and make them accessible?
-  AbortList<PathLimitReached, EndOfWorldReached, ParticleStopped> abortList;
+  AbortList<PathLimitReached, EndOfWorldReached> abortList;
```

Comment:
Not needed anymore?

Reply:
the case with `p=0` would result in a FPE so I dropped it for now